contrib/Makefile.am

```diff
@@ -1,2 +1,5 @@
-SUBDIRS = bash_completion.d bpftrace dracut initramfs pyzfs zcp
```

Comment:
You've accidentally dropped `bpftrace` here.

Reply:
I put it back.